### PR TITLE
Build the brand store takeover

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Built the "Brand store" takeover

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Refresh the homepage until you see the brand store page
- Check it matches [the design](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/174#issuecomment-453536750)
- Check the [copy is correct](https://docs.google.com/document/d/1P9iXXka1qGS1D_n14CiraIzIE-NDmMXMxQyM8v4A0T0/edit#)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4550

### CODE REVIEW

File: templates/takeovers/_snapcraft-brand-store_takeover.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <p class="p-heading--four">Control what is offered to your customers and avoid timely and expensive custom builds with Canonicals brand store offering.</p>
+        <p class="u-hide--small">
+          <a href="https://www.brighttalk.com/webcast/6793/346324?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_IOT_BrandStore_WBN_BrandStoreIntro" class="p-button--neutral u-no-margin--bottom">
+            <span class="p-link--external">Register for the webinar</span>
+          </a>
+        </p>
+      </div>
+      <div class="col-5">
+        <div class="u-align--center u-sv3 u-vertically-center">
+          <img src="https://assets.ubuntu.com/v1/44fd30d7-Snap+Store+shopping+icon+darkhandle.svg" class="p-takeover__image" alt="" />
```
Comment (deadlight) [inline templates/takeovers/_snapcraft-brand-store_takeover.html]: Use the asset server template tag

File: templates/takeovers/_snapcraft-brand-store_takeover.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    <style>
+      .p-takeover {
+        background-blend-mode: color;
+        background-image: linear-gradient(134deg, #2D6162 0%, #83BFA1 100%);
+        background-color: #2D6162;
+      }
+      @media only screen and (min-width: 768px) {
+        .p-takeover__suru {
+          background-image: url('https://assets.ubuntu.com/v1/9a0310e3-suru-right-snapstore.svg');
```
Comment (deadlight) [inline templates/takeovers/_snapcraft-brand-store_takeover.html]: You can use the asset server template tag here too

**Comment by kwm14** [comment]
## Design review
Build matches design but are we able to increase the size of the Snapcraft infographic? it looks quite small in comparison to copy on the left and @anasereijo design.

Also copy hasn't been finalised as per Alex's comment, so will hold up on adding Design +1 😉 

_"Thanks Ana, sorry for the late reply, I've been away. designs look great. Will work on getting the copy over to you."_

**Comment by Greggless** [comment]
# [Copy doc](https://docs.google.com/document/d/1P9iXXka1qGS1D_n14CiraIzIE-NDmMXMxQyM8v4A0T0/edit)

**Comment by anthonydillon** [comment]
@deadlight thanks done.

@kwm14 awaiting copy to be signed off

**Comment by kwm14** [comment]
## Design review
Takeover looks good with @Greggless updated cop, infographic and copy is better balanced inside the strip 👍 

screenshot 2019-01-23 at 09 39 53

**Comment by kwm14** [comment]
LGTM 👍

**Pull request merged**